EpikCord/client/http.py

```diff
@@ -215,6 +248,22 @@ async def request(
         await bucket.wait()
 
         for _ in range(5):
+            if json and not files:
+                kwargs["json"] = json
+            else:
+                form = aiohttp.FormData()
+                if json:
+                    form.add_field("payload_json", self.session.json_serialize(json))
+                if files:
+                    for i, file in enumerate(files):
+                        form.add_field(
+                            f"files[{i}]",
+                            file.contents,
+                            filename=file.filename,
+                            content_type=file.mime_type,
+                        )
+                kwargs["data"] = form
```

Comment:
Should be extracted

Reply:
That's a bit much? It's just a for loop adding fields. It's ~5 lines unformatted.

Reply:
It's not about lines being "unformatted" it's about having a well chuncked codebase with smaller function that are easier to maintain and does a single thing